spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -124,6 +124,16 @@
     then_I_see_a_page_reverted_success_notice
   end
 
+  scenario "User publishes and then makes more changes to a step by step page" do
+    given_there_is_a_step_by_step_page_assigned_to_me
+    and_I_visit_the_publish_page
+    and_I_publish_the_page
+    and_there_should_be_a_change_note "Minor update published by #{stub_user.name}"
```

Comment:
`then_there_should_be_a_change_note` (line 134 too)

Reply:
I'll add an alias method and fix up.